compiler/crates/graphql-transforms/src/transform_connections.rs

```diff
@@ -125,6 +125,7 @@ impl<'s> ConnectionTransform<'s> {
             for arg in &connection_directive.arguments {
                 if arg.name.item == DEFER_STREAM_CONSTANTS.if_arg
                     || arg.name.item == DEFER_STREAM_CONSTANTS.initial_count_arg
+                    || arg.name.item == DEFER_STREAM_CONSTANTS.initial_count_arg_oss
```

Comment:
I don't think this is the right approach. It should be either, not both. Have a look at how i'm currently supporting fb/oss for connection things.

https://github.com/facebook/relay/pull/3112/files#diff-86ef1b8ad53dcd24d2a88fabf01a1fc7R40

There is a few issues with it though, so still ironing out the kinks

Reply:
My assumption is the schema you are working with is not likely to support both. In this case if you did pass both `initialCount` and `initial_count` to a `@stream_connection` directive, you will get a schema validation error that one of those args is invalid.

Reply:
No sorry. Its more that _both_ shouldn't be supported at the same time. As the spec _will_ dictate the oss variant should be used. All I was saying, is to instead of _or_ the 2, put it behind a compiler flag, such that OSS people only get the spec version.

Reply:
Agree here: rather than have the code allow either argument name, this would ideally be a compiler config (similar to what we do for connections, which also have different naming internally and in OSS). Then the code would just check for that single name.

Reply:
@josephsavona is the compiler config for the js compiler in the Relay repo? I only see the OSS names https://github.com/facebook/relay/blob/master/packages/relay-runtime/handlers/connection/ConnectionInterface.js#L55

Reply:
Yeah we have an internal version of that file too, which uses FB's naming (basically the same but snake case). I think we can do the same thing, just put all the constants in a file similar to ConnectionInterface, and then we can maintain an internal version.

Reply:
@josephsavona I updated the PR to add a config to the JS compiler similar to the ConnectionInterface. '

The `hasNext` => `extensions.is_final` is still a runtime check, since it is not a just a property name change (and the check is in relay-runtime not the compiler).

I also removed the rust change. @maraisr maybe you can help out and add it to your pr https://github.com/facebook/relay/pull/3112?

Reply:
See 1386365510126b337a1409dd2a7ff11dbe895f66 for how we can add this to the config in the Relay Rust compiler.